ghpc-kenya-hiv-map/css/map.css

```
@@ -0,0 +1,228 @@
+  text-align: center;
+}
+
+.leaflet-popup-content-wrapper,
+.leaflet-popup-tip {
+  background-color: #e4e3e3;
+  box-shadow: 0 3px 14px rgba(0, 0, 0, 0.4);
+  border-radius: 3px;
+}
+
+@media screen and (max-width: 500px){
+  .toolbox {
+    max-width: 80% !important ;
+  }
+}
+
+.legend {
+  position: relative;
+  top: .313rem;
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  padding-bottom: .75rem;
+  font-size: .75rem;
+}
+
+.legend[data-type="total_cases"] {
+  --ramp1: #ffcfbf;
+  --ramp2: #fd998c;
+  --ramp3: #e56b61;
+  --ramp4: #bc4741;
+  --ramp5: #942223;
+  --ramp6: #670000;
+}
+
+.legend li {
+  padding: 0px;
+  margin: 0;
+  position: relative;
+}
+
+.legend li::before {
+  content: "";
+  position: relative;
+  padding: 0;
+  margin: 0;
+  display: inline-block;
+  width: 25px;
+  height: 5px;
+  background: var(--bg);
+}
+
+#ramp1::before {
+  background-color: var(--ramp1);
+  border: 3px solid var(--ramp1);
+}
+
+#ramp2::before {
+  background-color: var(--ramp2);
+  border: 3px solid var(--ramp2);
+}
+
+#ramp3::before {
+  background-color: var(--ramp3);
+  border: 3px solid var(--ramp3);
+}
+
+#ramp4::before {
+  background-color: var(--ramp4);
+  border: 3px solid var(--ramp4);
+}
+
+#ramp5::before {
+  background-color: var(--ramp5);
+  border: 3px solid var(--ramp5);
+}
+
+#ramp6::before {
+  background-color: var(--ramp6);
+  border: 3px solid var(--ramp6);
+}
+
+.legend__value {
+  position: relative;
+}
+
+.legend__value > span {
+  position: absolute;
+  top: .938rem;
+  right: 1.438rem;
+}
+
+.menu-translate {
+  position: relative;
+  padding-left: 10px;
+  color: #2e3c43;
+  font-weight: 800;
+  font-size: 22px;
+  line-height: 22px;
+  font-family: "Montserrat", Arial, Helvetica, sans-serif;
+}
+
+.hamburger {
+  position: relative;
+  width: 100%;
+  top: 50px;
+  left: 24px;
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 4px;
+  padding: 16px;
+  margin-left: -24px;
+  margin-right: -24px;
+}
+
+.hamburger span:nth-child(1) {
+  transform-origin: 0% 0%;
+}
+
+.hamburger span:nth-child(3) {
+  transform-origin: 0% 100%;
+}
+
+aside.toolbox .ui:checked ~ .hamburger span:nth-child(1) {
+  opacity: 1;
+  transform: rotate(45deg) translate(-3px, -3px);
+}
+
+aside.toolbox .ui:checked ~ .hamburger span:nth-child(2) {
+  opacity: 0;
+  transform: rotate(0deg) scale(0.2, 0.2);
+}
+
+aside.toolbox .ui:checked ~ .hamburger span:nth-child(3) {
+  transform: rotate(-45deg) translate(0, 0px);
+}
+
+.mobile-only { 
+  height:80px;
+  font:unset;
+  font-family:Open Sans,sans-serif;
+  font-size:12px;
+  visibility:visible
+}
+@media screen and (min-width:48em){
+  .mobile-only{
+    height:0;
+    visibility:hidden
+  }
+}
+
+hr {
+  background: #b3b3b3;
+  height: 1px;
+  border-width: 0;
+  color: gray;
+}
+
+.is-hidden {
+  display: none;
+}
+
+@media (max-width: 770px) {
+  .description {
+    padding-top: 10px;
+  }
+}
```

To control the div. So it doesn't overlap with the credits when the screen is resized. My suggestion is:
```suggestion
}

@media (max-width: 17.875rem) {
  #source__left {
    position: absolute;
    bottom: 40px;
    height: 50px;
    padding: 0 0.312rem;
  }
}

@media screen and (min-width: 17.937rem) and (max-width: 26.812rem) {
  #source__left {
    position: absolute;
    bottom: 40px;
    height: 36px;
    padding: 0 0.312rem;
  }
}

@media screen and (min-width: 26.875rem) and (max-width: 48.125rem) {
  #source__left {
    position: absolute;
    bottom: 20px;
    height: 26px;
    padding: 0 0.312rem;
  }
}

@media screen and (min-width: 48.187rem) and (max-width: 76.875rem) {
  #source__left {
    position: absolute;
    bottom: 20px;
    height: 24px;
    padding: 0 0.312rem;
  }
}

```